content/learn/case-studies/transunion.md

```diff
@@ -0,0 +1,50 @@
+---
+$order: 26
+$title: TransUnion
+$date: Mar 12, 2018
+$category: ecommerce
+$parent: /content/learn/case-studies.html
+class: case-study post-detail
+
+components:
+  - social-share
+
+headline: TransUnion scores more conversions with faster page loads via AMP
+results:
+    - stat: 3%
+      description: increase in conversion
+    - stat: 26% 
+      description: lower bounce rate
+    - stat: 2.5x
+      description: time on site
+
+logo:
+  src: "case-studies/transunion_logo.png"
+  width: 230
+  height: 65
+download: "case-studies/transunion.pdf"
```

Comment:
The actual PDF file is the same page duplicated.  I think it should only be a single page.  Can you correct please.

Reply:
Ah, yes thanks for catching this. Meant to delete the other page (we swapped phone shells).

Reply:
[transunion.pdf](https://github.com/ampproject/docs/files/1891259/transunion.pdf)